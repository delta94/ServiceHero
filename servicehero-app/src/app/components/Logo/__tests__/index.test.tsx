import React from 'react';
import { render } from '@testing-library/react';
import { Logo } from '..';
import { BrowserRouter } from 'react-router-dom';

describe('<Logo  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(
      <BrowserRouter>
        <Logo />
      </BrowserRouter>,
    );
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
