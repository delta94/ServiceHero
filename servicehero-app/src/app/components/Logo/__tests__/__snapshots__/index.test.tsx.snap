// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo  /> should match snapshot 1`] = `
.c1 {
  color: white;
}

.c2 {
  font-weight: bold;
}

.c0 {
  width: 200px;
  font-size: 1.4em;
}

<a
  class="c0"
  href="/"
>
  <span
    class="c1"
  >
    Service
  </span>
  <span
    class="c2"
  >
    Hero
  </span>
</a>
`;
